### PR TITLE
catch error on regular run

### PR DESCRIPTION
**Pull request by enoch85**

### CODE REVIEW

**Comment by enoch85** [comment]
@szaimen sorry, now I remember why we made it like that - Shellcheck!

File: lets-encrypt/activate-tls.sh [hunk 1]
```diff
@@ -220,44 +220,34 @@ if [ -n "$DEDYNDOMAIN" ]
...
-    certonly
-else
-    generate_cert "$TLSDOMAIN"
-fi
-
-# Generate DHparams
-if [ -d "$CERTFILES" ]
-then
-    if [ ! -f "$DHPARAMS_TLS" ]
+    if certbot certonly --manual --text --rsa-key-size 4096 --renew-by-default --server https://acme-v02.api.letsencrypt.org/directory no-eff-email --agree-tos --preferred-challenges dns --manual-auth-hook "$SCRIPTS"/deSEC/hook.sh --manual-cleanup-hook "$SCRIPTS"/deSEC/hook.sh -d "$DEDYNDOMAIN"
```
Comment (szaimen) [inline lets-encrypt/activate-tls.sh:223]: Why did you put all of this into one line? It is much harder to read like this, imho...

Comment (enoch85) [reply lets-encrypt/activate-tls.sh:223]: to do the `if` statement.

Comment (szaimen) [reply lets-encrypt/activate-tls.sh:223]: But this is also possible with using many lines. See for example: https://github.com/nextcloud/vm/blob/277908e21a2b48891c5e4a5f7c357c85e32b914c/not-supported/borgbackup.sh#L369-L370

Comment (szaimen) [reply lets-encrypt/activate-tls.sh:223]: ping @enoch85

Comment (enoch85) [reply lets-encrypt/activate-tls.sh:223]: Feel free to create a PR where that's sorted.

**Pull request merged**